daemon/mgr/image.go

```diff
@@ -128,14 +128,37 @@ func (mgr *ImageManager) GetImage(ctx context.Context, idOrRef string) (*types.I
 }
 
 // RemoveImage deletes an image by reference.
-func (mgr *ImageManager) RemoveImage(ctx context.Context, image *types.ImageInfo, option *ImageRemoveOption) error {
-	if err := mgr.client.RemoveImage(ctx, image.Name); err != nil {
+func (mgr *ImageManager) RemoveImage(ctx context.Context, image *types.ImageInfo, name string, option *ImageRemoveOption) error {
```

Comment:
Why we add new `name` params, can we use `image.Name` ?

Reply:
To keep consistency with Moby, the `Name`  field does not exist in `imageInfo`. So i am afrid we can not use `image.Name`.